### ISSUE TITLE
Transfer Learning for Estimation of Pendubot Angular Position Using Deep Neural Networks. (arXiv:2201.12649v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#15312]
Transfer Learning for Estimation of Pendubot Angular Position Using Deep Neural Networks. (arXiv:2201.12649v1 [cs.CV])<br>
https://ift.tt/wOrbzB8ep<br>
<p>In this paper, a machine learning based approach is introduced to estimate Pendubot angular position from its captured images. Initially, a baseline algorithm is introduced to estimate the angle using conventional image processing technique. The baseline algorithm performs well for the cases that the Pendubot is not moving fast. However, when moving quickly due to a free fall, the Pendubot appears as a blurred object in the captured image in a way that the baseline algorithm fails to estimate the angle. Consequently, a Deep Neural Network (DNN) based algorithm is introduced to cope with this challenge. The approach relies on the concept of transfer learning to allow the training of the DNN on a very small fine-tuning dataset. The base algorithm is used to create the ground truth labels of the fine-tuning dataset. Experimental results on the held-out evaluation set show that the proposed approach achieves a median absolute error of 0.02 and 0.06 degrees for the sharp and blurry images respectively.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
